compiler/one-cmds/CMakeLists.txt

```diff
@@ -74,6 +74,15 @@ foreach(ONE_UTILITY IN ITEMS ${ONE_UTILITY_FILES})
 
 endforeach(ONE_UTILITY)
 
+# one-pack internally uses model2nnpkg tool
+set(MODEL2NNPKG "${NNAS_PROJECT_SOURCE_DIR}/tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh")
+install(FILES ${MODEL2NNPKG}
+              PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE
+                          GROUP_READ GROUP_EXECUTE
+                          WORLD_READ WORLD_EXECUTE
+              DESTINATION bin
+              RENAME "model2nnpkg")
```

Comment:
If we don't change name, we don't need to modify `compiler/one-cmds/one-pack` and `infra/debian/compiler/one-compiler.install`, and `infra/packaging/res/tf2nnpkg.20221125` files.